frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -166,8 +166,8 @@ if Device:isAndroid() then
         }
     end
 
-    -- fullscreen toggle on devices with compatible fullscreen methods (apis 14-16)
-    if Device.firmware_rev <= 16 then
+    -- fullscreen toggle on devices with compatible fullscreen methods (apis 14-18)
+    if Device.firmware_rev < 19 then
```

Comment:
Maybe a little info message that a restart is required for changes to take effect?

Reply:
done, but only for jelly bean apis.